src/gui/ipsubnetwhitelistoptions.ui

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>IPSubnetWhitelistOptions</class>
+ <widget class="QDialog" name="IPSubnetWhitelistOptions">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>360</width>
+    <height>450</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>List of whitelisted IP subnets</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QFrame" name="whitelistedIPSubnetBox">
+     <property name="enabled">
+      <bool>true</bool>
+     </property>
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
```

Comment:
the default works as well, pls remove it.
IMO, only property value that is different than the default should be saved, otherwise you won't know if it's really needed.
There is a small arrow on the right side of property setting in Qt Designer, use it to reset to default.
And please recheck all those properties that are explicitly set.